config/application.rb

```diff
@@ -87,5 +90,9 @@ class Application < Rails::Application
       end
 
     end
+
+    config.after_initialize do
```

Comment:
Well the important thing is that the application doesn't start when there are wring settings. And some of the initializers are for configuration and need to be validated as well. So I use `after_initialize` to catch those initializers as well.